ChaosMod/Effects/db/Player/PlayerFPS.cpp

```diff
@@ -0,0 +1,34 @@
+/*
+	Effect By OnlyRealNubs
+*/
+
+#include "stdafx.h"
+static int scaleform;
+
+static void OnStart()
+{
+	scaleform = REQUEST_SCALEFORM_MOVIE("MP_BIG_MESSAGE_FREEMODE");
+	while (!HAS_SCALEFORM_MOVIE_LOADED(scaleform))
+	{
+		WAIT(0);
+	}
+}
+
+static void OnTick()
+{
+	float fps = (1000 / GET_FRAME_TIME()) / 1000;

```

Comment:
"0.x Gamespeed" multiplies this value. Also showing the weapon wheel or holding alt increases the counter times 10 or something like that